src/urdf/Urdf.js

```diff
@@ -118,3 +118,7 @@ ROS3D.Urdf = function(options) {
   }
 };
 ROS3D.Urdf.prototype.__proto__ = THREE.Object3D.prototype;
+
+ROS3D.Urdf.prototype.unsubscribeTf = function () {
+  this.children.forEach(function(n) { n.unsubscribeTf(); });
```

Comment:
You may want to validate that the child objects are infact `SceneNodes` or atleast has the `unsubscribeTf` function before calling a potentially undefined method as any custom component attached to the `Object3D` would be returned here.